src/components/RecentProtocols/index.tsx

```diff
@@ -255,15 +252,14 @@ export function RecentProtocols({ title, name, header, protocols, chainList, for
 					tvlPrevMonth,
 					change_1d: getPercentChange(tvl, tvlPrevDay),
 					change_7d: getPercentChange(tvl, tvlPrevWeek),
-					change_1m: getPercentChange(tvl, tvlPrevMonth),
-					listedAt: Number(((currentTimestamp - p.listedAt) / secondsInDay).toFixed(2))
```

Comment:
now we don't do this conversion here, it's not needed, as we now rely on pure dates rather than days with decimals